metaschema-cli/src/main/java/gov/nist/secauto/metaschema/cli/commands/AbstractValidateContentCommand.java

```diff
@@ -236,9 +236,9 @@ public ExitStatus execute() {
           // this case was already checked for
           return ExitCode.INPUT_ERROR.exitMessage("The provided source file '" + source + "' does not exist.");
         } catch (IOException ex) {
-          return ExitCode.FAIL.exit().withThrowable(ex);
+          return ExitCode.PROCESSING_ERROR.exit().withThrowable(ex);
         } catch (IllegalArgumentException ex) {
-          return ExitCode.FAIL.exitMessage(
+          return ExitCode.INPUT_ERROR.exitMessage(
```

Comment:
Minor, non-blocking. If you like the above change:

```suggestion
          return ExitCode.IO_ERROR.exitMessage(
```